src/azul/service/avro_pfb.py

```diff
@@ -702,11 +719,20 @@ def _update_replica_schema(*,
 
 def _new_replica_schema(*,
                         path: Sequence[str],
-                        value: AnyMutableJSON,
+                        value: AnyJSON,
                         ) -> AnyMutableJSON:
+    """
+    Create a part of a PFB schema to describe a part of a PFB entity represented
+    as a JSON document.
+    :param path: the location of `value` within the root document as a series
+                 of keys. The first key should be the name of the underlying PFB
+                 entity's type within the schema.
+    :param value: a part of a PFB entity.
+    :return: JSON describing the contents of `value` as a part of PFB schema.
+    """
     if value is None:
         result = 'null'
-    elif isinstance(value, list):
+    elif isinstance(value, (tuple, list)):
```

Comment:
Please explain the need for handling `tuple` as it would be confusing if that case never actually occured. `Sequence` is the best approximation of an immutable list in Python but that doesn't mean we need to actually handle `tuple` as a concrete case.

Reply:
We do not need to handle tuples. I was just aiming for consistency the updated type annotation of the `value`  parameter.

Reply:
The presence of a tuple in a JSON structure would indicate an anomaly, so we should be strict here as the overriding principle.